packages/payments/README.md

```diff
@@ -0,0 +1,14 @@
+<p align="center">
+<br />
+<a href="https://thirdweb.com"><img src="https://github.com/thirdweb-dev/js/blob/main/packages/sdk/logo.svg?raw=true" width="200" alt=""/></a>
```

Comment:
incorrect path (same in the rest of this file)

Reply:
Updated the npm paths, left the github / asset paths unchanged